### DISCUSSION TITLE
SCWindow坐标的一些转换技巧 | 奇风岁月

### DISCUSSION THREAD

**Discussion by giscus** [travisbikkle/travisbikkle.github.io#5] [General]
# SCWindow坐标的一些转换技巧 | 奇风岁月

SCWindow坐标的一些转换技巧

https://travisbikkle.pages.dev/2024/10/how-did-i-develop-mactoys-pin-window/

**Comment by travisbikkle** [comment]
已经上架了哦 [MagicToy on AppStore](https://apps.apple.com/us/app/magictoy/id6737158347) 免费试用，或访问 https://mactoys.eusoftbank.com 了解更多 😄
